README.md

```
@@ -1,3 +1,134 @@
+<<<<<<< HEAD
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset=".github/mark-dark.svg">
+  <img    loading="lazy"     style="max-width: 120px; float: right; margin: 0 0 20px 20px;"
+  alt="ByteBeats official logo" align="right" width="120" src="./assets/bytebeastslogo.svg">
+</picture>
+
+<a href="https://x.com/0xByteBeasts">
+<img src="https://img.shields.io/twitter/follow/0xByteBeasts?style=social"/>
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)